frontend/src/app/supportAdmin/DeviceType/DeviceForm.test.tsx

```diff
@@ -1,24 +1,91 @@
 import { render, screen, within } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 
-import ManageDevicesForm from "./ManageDevicesForm";
-import { addValue } from "./DeviceTypeForm.test";
```

Comment:
importing a method from FileA.test.tx into FileB.test.tx seemed to cause running FileB to trigger running all tests in FileA as well, which was throwing off my snapshots, so now all 3 frontend test files have their own `addValue` method. curious if anyone knows what that's about or if I should make a testutils or something